src/main/java/com/exasol/sql/ValueTableRow.java

```diff
@@ -108,6 +115,17 @@ public Builder add(final int... values) {
             return this;
         }
 
+        /**
+         * Add one or more double literals to the row.
+         *
+         * @param values doubles to be added
+         * @return <code>this</code> for fluent programming
+         */
+        public Builder add(final double... values) {
+            stream(values).mapToObj(DoubleLiteral::of).forEach(this.expressions::add);
```

Comment:
Please use the same way as shown above.
While Streams are useful, they are slower in this scenario and the code is harder to read.

Reply:
Sonar Cloud analysis failed for reasons that are not clear. Let's investigate this together.

Reply:
:( okay